libs/ui/design-system/src/lib/checkbox.tsx

```diff
@@ -1,17 +1,31 @@
-import { Checkbox as CheckboxPrimitive } from '@nmit-coursition/ui/primitives/checkbox'
+import * as CheckboxPrimitive from '@nmit-coursition/ui/primitives/checkbox'
 import { Label } from '@nmit-coursition/ui/primitives/label'
+import { cn } from '@nmit-coursition/ui/utils'
+import { type VariantProps, cva } from 'class-variance-authority'
 
-interface CheckboxProps {
+const checkboxVariants = cva('', {
+  variants: {
+    variant: {
+      primary: '',
+      outlined: '',
```

Comment:
**suggestion:** Same thing as before, there's no `outlined` variant, so `Checkbox` probably has no variants at all. It's `focused` state.